pkg/printers/template.go

```diff
@@ -112,3 +116,11 @@ func (p *TemplatePrinter) safeExecute(w io.Writer, obj interface{}) error {
 	}
 	return retErr
 }
+
+func base64decode(v string) string {
+	data, err := base64.StdEncoding.DecodeString(v)
+	if err != nil {
+		return err.Error()
```

Comment:
this isn't good… it means the output from an error is indistinguishable from success

Reply:
Make this a two value return:

> Each function must have either a single return value, or two return values of which the second has type error. In that case, if the second (error) return value evaluates to non-nil during execution, execution terminates and Execute returns that error.

Reply:
Thanks! Will do.

Reply:
I also figured out how to trigger the error path in the test using an intentionally-broken [Marshaler](https://golang.org/pkg/encoding/json/#Marshaler).